uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/sdk/AuthResult.java

```diff
@@ -0,0 +1,68 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.ui.automation.sdk;
+
+//import com.microsoft.aad.adal.AuthenticationResult;
+import android.text.TextUtils;
+import org.junit.Assert;
+import androidx.annotation.NonNull;
+import lombok.Getter;
+
+/**
+ * An abstract wrapper class for the result obtained from acquire token interactively
+ * or silently, it wraps all the parameters obtained from token and also
+ * the exception, with methods to assert success or failures.
+ */
+@Getter
+public abstract class AuthResult {
+
+    protected static String accessToken;
+    protected static String idToken;
+    protected static String userId;
+    protected static String username;
+    protected static String authority;
+    protected static Exception exception;
```

Comment:
Why `static`? and why `protected? why not `private`?

Reply:
These should really not be static

Reply:
I made them static so that I can use them in class easily for testing purpose but yes we shouldn't use static, so I will be removing them other then that I made them protected so that parent class can access these variables.